pom.xml

```diff
@@ -42,6 +42,22 @@
                         </configuration>
                     </execution>
                 </executions>
+                <configuration>
+                    <relocations>
+                        <relocation>
+                            <pattern>fr.mrmicky.fastinv</pattern>
```

Comment:
No clue what this is.

Reply:
This is an inventory API very light weight and better than using Bukkit directly.